### ISSUE TITLE
Login not working

### ISSUE THREAD

**Issue by GiacomoK** [webvimark/user-management#115]
I have a problem with login. I installed this component, and everything worked fine.

The problem is that when I login, I actually pass credentials check, but still I am not logged in Yii.

Anybody knows what can I do?

Thanks.

**Comment by webvimark** [comment]
Hi, need more data =) Did it redirect you or you stays on the same page ?

**Comment by GiacomoK** [comment]
I tried both. I tried to redirect after login or to return to homepage (I have a clean Yii installation).

The yii::app->user->IsGuest line always returns true.

**Comment by webvimark** [comment]
No, I mean after you enter credential and click "Login" button - application redirects you to another page or stays there ?

**Comment by GiacomoK** [comment]
it redirects in the homepage, but I am not logged.

**Comment by webvimark** [comment]
Lets debug :)

Could you set breakpoints (like die('1'), die('2'), etc) in AuthController on the following lines

- 44 ( die('1'); before return $this->goHome(); )
- 57 ( die('2'); before return $this->goBack(); )

**Comment by GiacomoK** [comment]
it dies at second die (line 57).

**Comment by GiacomoK** [comment]
Anyone can help me?

**Comment by webvimark** [comment]
Hmm, it's really strange. Like **really** strange. Could you upload your app in some repository. Just remove all data except core and this moduel and I'll try find what is the problem

**Comment by marcelodeandrade** [comment]
Have the same problem, any solution?

**Comment by marcelodeandrade** [comment]
@GiacomoK  Check in backend config and remove the 'user' component default. Solved my problem.

**Comment by HanafiAhmat** [comment]
@GiacomoK , if you installed Yii2-advance-app, follow @marcelodeandrade 's recommendation. Seems like the configuration instructions for this module is based on Yii2-basic-app.

**Comment by msankar1991** [comment]
@HanafiAhmat , I've followed above steps given but it not works for yii2 advanced app. After login redirects to login page only. May I know how to solve it.

**Comment by marcelodeandrade** [comment]
@msankar1991 You follow my tip? Check all configs [common,frontend,backend]

**Comment by msankar1991** [comment]
@HanafiAhmat , Yes. I've followed the tip for remove user component default only but it not works.

**Comment by HanafiAhmat** [comment]
@msankar1991 , if your advance app is a clean install, @marcelodeandrade tip should work. Else, there might be some misconfigurations in either your common config or backend config. can you show us your configuration files?

**Comment by Sabirgojayev** [comment]
Have similar problem. When I log in from backend I am not logged in frontend

**Comment by HanafiAhmat** [comment]
@sabirgojayev, does your frontend app share the same cookie session with backend app? Default yii2-advanced-app configurations separates them.

**Comment by Sabirgojayev** [comment]
I solved the problem was as @HanafiAhmat said in cookie session parameter in config
